docs/docs/examples/guides/isaaclab-aica-bridge.md

```diff
@@ -0,0 +1,219 @@
+- **headless**: When set to `true`, the simulator runs in headless mode, which is useful for running simulations without
+  a graphical interface.
+- **device**: Specifies the device to use for simulation. The default is `cuda`, which utilizes GPU acceleration. You
+  can change this to `cpu` if you want to run the simulation on the CPU instead.
+
+Ensure these parameters are correctly configured to enable seamless communication between the simulator and your AICA
+application. In case you want to run the simulator with different parameters, you can do so by running the following
+command in the `run_bridge.py` script:
+
+```shell
+python3 scripts/custom/aica_bridge/run_bridge.py --scene <your_scene_name> --rate <simulation_rate> --end_effector <end_effector_link_name> --force_sensor <true/false> --state_port <state_port> --command_port <command_port> --force_port <force_port> --command_interface <position/velocity> --headless <true/false> --device <cuda/cpu>
+```
+
+## Creating an AICA Application
+
+To get started, follow these steps to create a new application in **AICA Studio**:
+
+1. Open **AICA Launcher** and launch a configuration using the latest core image along with the latest Universal Robots
+   collection.
+2. Click the **New Application** button located in the top-left corner of the **AICA Studio** interface.
+3. In the new application, add the following components:
+
+   - **Signal Point Attractor**: Drives the robot’s movement toward a specified target frame.
+   - **Frame to Signal**: Converts the target frame into a signal that feeds into the point attractor.
+   - **Hardware Interface**: Connects your application to the Isaac Lab simulator, allowing you to control the UR5e
+     robot and receive real-time state updates.
+   - **IK Velocity Controller**: Calculates the inverse kinematics for the UR5e, enabling smooth motion toward the
+     target frame.
+
+4. Connect the events and signals as shown in the image. Make sure to auto-configure and auto-activate all components
+   and controllers.
+
+<div class="text--center">
+  <img src={application} alt="Signal Point Attractor Application" />
+</div>
+
+5. Press **Play**, then switch to the **3D Visualizer** tab in **AICA Studio**. You should see the UR5e robot in the
+   scene. Use the **Record Frame** button to capture a target frame at the robot's tool, specify the tool frame name
+   (for UR robots from the collection, that is `ur_tool0`), and then move the frame to your desired target position.
+6. Once the target frame is set, parametrize the **Frame to Signal** component with the name of the target frame you
+   just created.
+
+Now that you have set up the application, let's go over the necessary steps that needs to be done on the Hardware
+Interface level to connect **AICA System** to the Isaac Lab simulator.
+
+## Configuring the Hardware Interface
+
+The **Hardware Interface** serves as the communication bridge between the **AICA System** and external hardware or
```

Comment:
I still think this needs to come before we create the application, because the screenshot above already assumes this URDF is present